Added my advice link

Luilu27 opened the pull request.

twosavoie commented on index.html:
```diff
@@ -18,8 +18,11 @@ <h2>Our Advice for Newbies!</h2>
     <a href="advice/angela-rivera-advice.html" target="_blank">Angela Rivera's Advice 💙</a><br>
     <a href="advice/aileen-cm-advice.html">Aileen CM's advice!🇵🇷</a><br>
     <a href="advice/destiny1s-advice.html">🤍Destiny's Advice🤍</a><br>
-    <a href="advice/torres-ml-advice.html" target="_blank"
```
HI @Luilu27,

Congrats on issuing your first pull request! That's awesome!!

It looks like the styling Mari added to the link to her advice page has been removed. Can you add it back? (It's the link just above your link.)

Also, there are two added <i> tags. Is this something you want added to your link? If so, can you add it between the anchor tags `<a></a>` of the link to your advice page?

Just in case you're adding Font Awesome icons, since there isn't a link to Font Awesome in the <head> of the index.html file, the icon isn't going to render. If you would like to add a Font Awesome icon to your advice page - that's perfectly fine! You can add the link to Font Awesome in the <head> of _your_ advice file and add a Font Awesome icon to the content of your page. (Or you can add an emoji instead like some other Skillcrushers have done.)

Once you've made those two changes, add and commit them. Then push the advice branch to GitHub. It will automatically update your pull request and we'll take a look!

Reach out if you have any questions!

Best,
Lisa

twosavoie commented:
Hi @Luilu27,

Just checking to see if you have any questions about this pull request. 

Just in case it's helpful, you can either copy and paste Mari's link to her advice file from the Skillcrush repo [here](https://github.com/skillcrush/git-practice-v2/blob/main/index.html). 

Or you can fetch from the upstream repo, merge it into your main branch, merge main into advice, and resolve any conflcts. 

Then push your advice branch to GitHub to update your pull request.

Let us know if there's anything we can clarify!


Best,
Lisa

Luilu27 commented:
Hi Lisa,

Sorry for the late reply. Moving along the lesson, I realized we were asked
to work with our Jublile Austin project, and unfortunately, I got stuck on
the last part of the CSS. I started to work on that a bit to find any bugs
and refresh my memory, but again, I kept running into the same problem.

I decided that rather than switching subjects, I'm going to focus on one
for now and return to the CSS lesson at a later time.

So, saying this. Back to GIT.

I believe I have fixed Mari's link (*all the extra CSS on her link tag made
me think it was extra and not part of her tag*). I added it, committed it
as "added missing link," pushed it on GitBash, and pulled a request on
GitHub.

I got this message *screenshot below-
[image: image.png]

IS this correct?


On Tue, Nov 5, 2024 at 5:18 PM Lisa Savoie ***@***.***> wrote:

> Hi @Luilu27 <https://github.com/Luilu27>,
>
> Just checking to see if you have any questions about this pull request.
>
> Just in case it's helpful, you can either copy and paste Mari's link to
> her advice file from the Skillcrush repo here
> <https://github.com/skillcrush/git-practice-v2/blob/main/index.html>.
>
> Or you can fetch from the upstream repo, merge it into your main branch,
> merge main into advice, and resolve any conflcts.
>
> Then push your advice branch to GitHub to update your pull request.
>
> Let us know if there's anything we can clarify!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice-v2/pull/1#issuecomment-2457613648>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLJF2K7TFWRA65L2U3YUKM3Z7DVVBAVCNFSM6AAAAABRAFICWGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINJXGYYTGNRUHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Luilu27 commented:
When I tried to merge it into the main - I ran into a conflict.
I thought I had fixed the conflict, both on Visual Studio code and Git, but
when I pulled the request again - it showed the same conflict. I think I am
definitely doing something wrong or maybe repeating steps..???

On Fri, Nov 8, 2024 at 10:17 AM Lui C. ***@***.***> wrote:

> Hi Lisa,
>
> Sorry for the late reply. Moving along the lesson, I realized we were
> asked to work with our Jublile Austin project, and unfortunately, I got
> stuck on the last part of the CSS. I started to work on that a bit to find
> any bugs and refresh my memory, but again, I kept running into the
> same problem.
>
> I decided that rather than switching subjects, I'm going to focus on one
> for now and return to the CSS lesson at a later time.
>
> So, saying this. Back to GIT.
>
> I believe I have fixed Mari's link (*all the extra CSS on her link tag
> made me think it was extra and not part of her tag*). I added it,
> committed it as "added missing link," pushed it on GitBash, and pulled a
> request on GitHub.
>
> I got this message *screenshot below-
> [image: image.png]
>
> IS this correct?
>
>
> On Tue, Nov 5, 2024 at 5:18 PM Lisa Savoie ***@***.***>
> wrote:
>
>> Hi @Luilu27 <https://github.com/Luilu27>,
>>
>> Just checking to see if you have any questions about this pull request.
>>
>> Just in case it's helpful, you can either copy and paste Mari's link to
>> her advice file from the Skillcrush repo here
>> <https://github.com/skillcrush/git-practice-v2/blob/main/index.html>.
>>
>> Or you can fetch from the upstream repo, merge it into your main branch,
>> merge main into advice, and resolve any conflcts.
>>
>> Then push your advice branch to GitHub to update your pull request.
>>
>> Let us know if there's anything we can clarify!
>>
>> Best,
>> Lisa
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skillcrush/git-practice-v2/pull/1#issuecomment-2457613648>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BLJF2K7TFWRA65L2U3YUKM3Z7DVVBAVCNFSM6AAAAABRAFICWGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINJXGYYTGNRUHA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

twosavoie commented:
Hi @Luilu27,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

Best,
Lisa
